webviz_subsurface/plugins/_tornado_plotter_fmu/_plugin_ids.py

```diff
@@ -0,0 +1,39 @@
+from webviz_config.utils import StrEnum
+
+
+class PluginIds:
```

Comment:
If all unused `Ids`  below are removed - consider moving the remaining `Ids` into an `Id` class in `TornadoPlotterFMU` in `_plugin.py` and delete this file as circular imports does not seem to be an issue here. Store functionality is not passed to the views/view elements as far as I can see.